README.md

```diff
@@ -1,24 +1,89 @@
-# README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Rails recipe App
 
-Things you may want to cover:
+>Rails recipe app is a Ruby on Rails application that keeps track of all your foods, recipes, ingredients, and inventories.
 
-* Ruby version
 
```

Comment:
- [ ] It would be better if we record a video presentation of the project. Kindly find below the details needed for creating the video as seen in the screenshot below

![Screenshot from 2022-09-08 18-32-37](https://user-images.githubusercontent.com/46388113/189201459-d8445470-128a-4bea-aeb1-e5df62772c98.png)